### ISSUE TITLE
Theme and tile colors

### ISSUE THREAD

**Issue by Pupix** [haydenbleasel/favicons#91]
Hello! 

Would be nice to be able to specify the theme/tile color in the configuration object, this way I don't have to manually change the resulting meta tags after the build process has completed.

Cheers.

**Comment by strarsis** [comment]
I thought the config key "background" offers this?

**Comment by haydenbleasel** [comment]
@strarsis It should. @Pupix is the background property not working for you?

**Comment by Pupix** [comment]
No, it's' not working.

With the following config 

```js
{
  appName: 'test-app',
  background: '#123456',
  path: 'app/icons/',
  display: 'standalone',
  orientation: 'portrait',
  version: 1.0,
  html: 'test/index.html',
  icons: {
    android: true,
    appleIcon: true,
    appleStartup: true,
    coast: true,
    favicons: true,
    firefox: true,
    opengraph: true,
    twitter: true,
    windows: true,
    yandex: true
  }
}
```

I get the following HTML lines:

```html
...
<meta name="theme-color" content="#FFFFFF">
<meta name="msapplication-TileColor" content="#FFFFFF">
...
```
 and this `manifest.json`:

```json
{
  "name": "expandjs-starter-kit",
  "display": "standalone",
  "orientation": "portrait",
  "icons": [...]
}
```

There's no hint of either [background](https://developer.mozilla.org/en-US/docs/Web/Manifest#background_color) or [theme color](https://developer.mozilla.org/en-US/docs/Web/Manifest#theme_color) in the manifest. I was expecting something like [this](https://github.com/PolymerElements/polymer-starter-kit/blob/master/app/manifest.json#L37).

**Comment by haydenbleasel** [comment]
Fixed in `favicons@4.4.5`
